cmd/vulndb/diffcmd.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"sort"
+
+	"github.com/facebookincubator/nvdtools/cvefeed"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(diffCmd)
+}
+
+func feedLoad(file string) (cvefeed.Dictionary, error) {
+	log.Printf("loading %s\n", file)
+	dict, err := cvefeed.LoadJSONDictionary(file)
+	if err != nil {
+		return nil, fmt.Errorf("failed to load dictionary %s: %v", file, err)
+	}
+	return dict, nil
+}
+
+func feedName(file string) string {
+	return filepath.Base(file[:len(file)-5])
+}
+
+func min(a, b int) int {
+	if a <= b {
+		return a
+	}
+	return b
+}
```

Comment:
Super-nitpick, but I'm not a fan of polluting package scope with names. You  seem to only use it once, you could've either do it inline, or define function in the soce of `printArraySorted`...